server/src/game/role/common_role.rs

```diff
@@ -134,21 +134,16 @@ pub(super) fn get_current_receive_chat_groups(game: &Game, actor_ref: PlayerRefe
     out
 }
 
+///Only works for roles connected to a faction/team
 pub(super) fn get_won_game(game: &Game, actor_ref: PlayerReference) -> bool {
```

Comment:
`get_team_won_game`

Reply:
Also, why does this only work for roles connected to a faction/team? Wouldn't this work just fine for arsonist?

Reply:
arsonist is a team. I need help naming this stuff

Reply:
'connected to end game conditions' here should probably say 'win based on their end game condition', unless this isn't the case. Either way this wording is strange